src/api/java/com/github/minecraftschurlimods/arsmagicalegacy/api/data/SpellPartDataProvider.java

```diff
@@ -155,7 +154,7 @@ public class SpellPartDataBuilder {
         private final float manaCost;
         private final Float burnout;
         private final List<ISpellIngredient> recipe = new ArrayList<>();
-        private final List<Either<Ingredient, ItemStack>> reagents = new ArrayList<>();
+        private final List<Ingredient> reagents = new ArrayList<>();
```

Comment:
this will remove the count from the reagent which is checked for in spell casting

Reply:
It won't, the ItemStack variant now just uses Ingredient.of(), which retains the count.

Reply:
does it tho? 
look at `com.github.minecraftschurlimods.arsmagicalegacy.common.spell.SpellHelper#hasReagents` and `net.minecraft.world.item.crafting.Ingredient#test`